lib/record_store/zone.rb

```diff
@@ -156,7 +160,8 @@ def validate_cname_records_for_same_fqdn
           when 'CNAME'
             errors.add(:records, "Multiple CNAME records are defined for #{record.fqdn}: #{record}")
           else
-            errors.add(:records, "A CNAME record is defined for #{cname_record.fqdn}, so this record is not allowed: #{record}")
+            errors.add(:records, "A CNAME record is defined for #{cname_record.fqdn}, "\
```

Comment:
Either: store the error message in a tmp var, OR line returns for the call.
I think the first option would be cleaner